docs/installation_configuration_guide/Configuration.md

```diff
@@ -18,13 +18,42 @@ Designer configuration  contains all settings for Nussknacker Designer - e.g. we
 One Nussknacker Designer deployment may be used to create various Scenario Types which:
                           
 * can be deployed with various [Deployment Managers](DeploymentManagerConfiguration.md)  to e.g. different Flink clusters 
-* Use different components and [model configurations](ModelConfiguration.md) 
+* use different components and [model configurations](ModelConfiguration.md) 
```

Comment:
"model configurations" -> "Model configurations", term Model is defined in the Glossary, we help readers notice that it is a special term

Reply:
:+1: